### PR TITLE
Fix seller/user swagger api

### LINKED ISSUE #101: Fix seller/user swagger and some sql detail improve
**Describe the bug**
swagger type error

### PR DESCRIPTION
**Pull request by x06lan**
Ex. Resolves #101 

### Changes
* swagger
    * seller insert tag
    * seller search tag
    * seller get report
* api
    * /seller/report
    * /seller/order
> [!CAUTION]
> # If  /seller/report have StatusInternalServerError maybe the data.json is miss the order_detail of order 10003
> ```
>  "order_detail":[{
>       "order_id": 10003,
>       "product_id": 10002,
>       "product_version": 1,
>       "quantity": 5
> }]
> ```
### Testing Approach (if unconventional)

Ex. Test Environment Setup: UNIX-like OS on PowerPC 601 Architecture
```sh
docker-compose up --build
```

### CODE REVIEW

**Comment by KatoToyoki** [comment]
i don't see the changes i mentioned before so here is the reminder : 
GET `/seller/product/{id}` lacks of "description"

File: pkg/router/seller/order.go
Comment (Error0229) [inline pkg/router/seller/order.go:1]: So there is still no info about who placed the order?

Comment (x06lan) [reply pkg/router/seller/order.go:1]: fixed

**Review by Error0229 — APPROVED** [review]
LGTM

**Review by NOOBDY — APPROVED** [review]

**Pull request merged**